### PR TITLE
add callbacktrigger to onshippingchange

### PR DESCRIPTION
**Pull request by andyfen**
needs a way to differentiate between address change and shipping option change

```
onShippingChange(data, actions) {
    if(data.callbackTrigger == 'SHIPPING_OPTION'){

    }
    if(data.callbackTrigger == 'SHIPPING_ADDRESS'){

    }
}
```

### CODE REVIEW

File: src/payment-flows/applepay/__tests__/initApplePaySession-shipping.test.js [hunk 1]
```diff
@@ -335,6 +335,7 @@ describe('initApplePay', () => {
             forceRestAPI:             true,
             orderID:                  'mock-orderID',
             partnerAttributionID:     'APPLEPAY',
+            callbackTrigger:          'SHIPPING_ADDRESS',
```
Comment (mnicpt) [inline src/payment-flows/applepay/__tests__/initApplePaySession-shipping.test.js:338]: @andyfen Can we make the triggers an enum?

Comment (andyfen) [reply src/payment-flows/applepay/__tests__/initApplePaySession-shipping.test.js:338]: i made it a constant i think that's what you mean ? happy to follow up restructuring where it goes later.

Comment (andyfen) [reply src/payment-flows/applepay/__tests__/initApplePaySession-shipping.test.js:338]: and this specific file is a test btw

Comment (mnicpt) [reply src/payment-flows/applepay/__tests__/initApplePaySession-shipping.test.js:338]: Either one is fine.

**Review by weihou — APPROVED** [review]

**Review by gregjopa — APPROVED** [review]

File: src/payment-flows/applepay/__tests__/initApplePaySession-shipping.test.js (hunk 1 above)
Comment (andyfen) [reply src/payment-flows/applepay/__tests__/initApplePaySession-shipping.test.js:338]: will follow up

**Pull request merged**